pages/national_resources/lu_resources.md

```diff
@@ -0,0 +1,86 @@
+---
+
+title: Luxembourg
+search_exclude: true
+country_code: LU
+contributors: [Pinar Alper, Marina Popleteeva, Vilem Ded, Wei Gu]
+coordinators: [Wei Gu]
+
+# Link to other pages in the tool assembly section on the RDMkit by listing the page_id 
+related_pages:
+  tool_assembly: [<!---REPLACE THIS with the page ID of the tool_assembly pages that you want to list here as related pages--->]
+
+training:
+  - name: Training in TeSS
+    registry: TeSS
+    url: https://tess.elixir-europe.org/nodes/luxembourg
+  - name: ELIXIR Luxembourg community in Zenodo
+    registry: Zenodo
+    url: https://zenodo.org/communities/elixir-lu/
+  - name: ELIXIR NODENAME YouTube
+  - registry: Youtube
+    url: <!--- URL of the channel --->
+  - name: <!---REPLACE THIS with the name of your training in registry or platform--->
+    registry: <!---REPLACE THIS with the name of the registry--->
+    url: <!---REPLACE THIS with the url of your training registry or platform--->
+
+# Refer to entries of the "main_tool_ and_resource_table" if institutions, organizations and projects from the country contribute to the development of international tools and resources. 
+ref_to_main_resources: 
+  -  FAIR Cookbook
+  -  DAISY
+  -  Data Catalog
+  -  <!---REPLACE THIS with the tool name--->
+
+# List here tools and resources mainly relevant for the specific country
+national_resources: 
```

Comment:
We might ask around to cover all resources. I can think about LLC and IBBL, but there is for sure more. Also PNED must be listed in future.

Reply:
PNED will be included as soon as web-site launched.
IBBL was discussed with Pinar and first answer was "probably not". We can re-consider.
What LLC offers for life science more than just a library? 
I forgot to include HPC.
P.s. OneDrive/Projects/Converge

Reply:
The page should imho provide more than just life science related resources. Basically any RDM related national services.

-  LLC to my knowledge had some attempts to provide also other services (for example instance of DMP online). But sure, right now it can be "only" library
- please check out [this](https://www.researchluxembourg.org/en/research-landscape/) resource. We may also refer to this web page as a whole.

Reply:
@vildead RDMkit and ELIXIR are life science focused resources.

Reply:
> @vildead RDMkit and ELIXIR are life science focused resources.

It was just a suggestion :) based on my quick review of other RDMKit National Pages - they mostly cover RDM in general.